assignment4.rb

```diff
@@ -0,0 +1,62 @@
+# Assignment 4
+# Accept the number from user and for that many times, calculate the Fibonacci series and print the whole output. Eg: 
+# If user enters 4 then output should be 1,1,2,3
+# If user enters 7 then output should be 1,1,2,3,5,8,13
+
+class Fibonacci
+ def fibo
+    puts "Enter Number"
+    number = gets.to_i
+    n1 = 0
+    n2 = 1
+    seq = []
+    seq.push(n1,n2)
+    n3 = n1 + n2
+
+    number.times do 
+    seq << n3
+    n1 = n2
+    n2 = n3
+    n3 = n2 + n1
+    end
+    number.times do |sequence|
+        puts seq[sequence]
+    end
```

Comment:
@monalis-cuelogic : What's the reason to have 2 `number.times` loop one below other ?